.github/workflows/tags.yaml

```diff
@@ -101,4 +101,4 @@ jobs:
           tag_name=$(basename $GITHUB_REF)
           hub release create "${assets[@]}" -m "$tag_name" "$tag_name"
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.CETE_GITHUB_TOKEN }}
```

Comment:
By default, this env is populated with the GitHub Actions token, that has the necessary permissions to release on the repository already, by changing this you will need to create a custom variable unnecessarily

Reply:
More on docs: https://docs.github.com/pt/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

Reply:
@vniche Thanks for confirming that.
I'll cancel this PR then.